analyzers/src/SonarAnalyzer.Shared/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -20,7 +20,7 @@
 
 namespace SonarAnalyzer.Extensions
 {
-    public static partial class SyntaxNodeExtensions
+    public static partial class SyntaxNodeExtensionsCSharp
```

Comment:
I am confused as to why the suffix is `CSharp` while we are in the Shared project.

Reply:
VS auto-rename.. the shared project creates quite some pain due to this